include/deal.II/base/utilities.h

```diff
@@ -812,13 +812,13 @@ namespace Utilities
     get_cpu_load();
 
     /**
-     * Return the current level of vectorization as described by
-     * DEAL_II_COMPILER_VECTORIZATION_LEVEL in vectorization.h as a string. The
+     * Return the instruction set extension for vectorization as described by
+     * DEAL_II_VECTORIZATION_WIDTH_IN_BITS in vectorization.h as a string. The
      * list of possible return values is:
      *
      * <table>
      * <tr>
-     *   <td><tt>VECTORIZATION_LEVEL</tt></td>
+     *   <td><tt>N_VECTORIZATION_BITS</tt></td>
```

Comment:
do you want to update this caption?

Reply:
I did, because I do not think the `VECTORIZATION_LEVEL` makes much sense now any more. I was a bit unsure if we should completely remove this column now that we have the name `VECTORIZATION_WIDTH_IN_BITS` and two columns with the same information.

Reply:
Sorry for my cryptic comment: I think you need to update this column name to reflect the rename to ``VECTORIZATION_WIDTH``.

Reply:
OK, I'll undo this change for now.